### DISCUSSION TITLE
Differences between Budgets and Categories/Tags

### DISCUSSION THREAD

**Discussion by jph** [firefly-iii/firefly-iii#5275] [Firefly III] [answered]
Hello. I've moved from a fairly strict envelope budget system, and whilst reading the documentation for Firefly III I noticed that the author indicated some transactions won't be part of a budget.

I'm having a bit of trouble reconciling this in my head considering my previous budgeting strategy where everything is enveloped.

Can anybody shed some more light on what I might not strictly 'budget' for, but still categorize for reporting purposes?

**Comment by JC5** [comment, marked as answer]
You can budget all your expenses. Not income, not transfers between accounts.

**Reply by jph** [reply]
> 
> 
> You can budget all your expenses. Not income, not transfers between accounts.

Thanks for the quick answer. Budgets are defined by/limited to a period aren't they? For example, it wouldn't make as much sense to use a budget for tracking a purchase that occurs once per quarter when my period is defined as monthly. I suppose I can see why simply categorizing a transaction like this for reporting purposes makes sense.
